### PR TITLE
Fix a bug when creating test data; if a community contained the strin…

### PR DESCRIPTION
**Pull request by peelman**
…g 'snmp', it would cause test data to not be generated.

Add line start/end anchors, and search for specific SNMP commands, instead of a snmp<wildcard>, and a couple of other minor tweaks...

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [X] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [X] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Review by Jellyfrog — APPROVED** [review]

**Comment by Jellyfrog** [comment]
Looks OK to me. I'll leave it to @murrant tho

File: LibreNMS/Util/ModuleTestHelper.php [hunk 1]
```diff
@@ -191,7 +191,7 @@ private function collectOids($device_id)
         $collection_output = preg_replace('/\033\[[\d;]+m/', '', $collection_output);
 
         // extract snmp queries
-        $snmp_query_regex = '/SNMP\[.*snmp(?:bulk)?([a-z]+)\' .+(udp|tcp|tcp6|udp6):(?:\[[0-9a-f:]+\]|[^:]+):[0-9]+\' \'(.+)\']/';
+        $snmp_query_regex = '/^SNMP\[\'.*snmp(?:bulk)?(walk|get|getnext)\' .+\'(udp|tcp|tcp6|udp6):(?:\[[0-9a-f:]+\]|[^:]+):[0-9]+\' \'(.+)\'\]$/m';
```
Comment (Jellyfrog) [inline LibreNMS/Util/ModuleTestHelper.php:194]: Hmm, I wonder if the start and end tags might be a problem, 
not all output is "clean", here is a line from a random poller run:
`ifMtu SNMP['/usr/bin/snmpbulkwalk' '-v2c' '-c' 'COMMUNITY' '-OQUst' '-m' 'IF-MIB' '-M' '/opt/librenms/mibs' 'udp:HOSTNAME:161' 'ifMtu']`

Comment (Jellyfrog) [inline LibreNMS/Util/ModuleTestHelper.php:194]: Also, this saves like 150 steps:
```suggestion
        $snmp_query_regex = '/SNMP\[\'.*?snmp(?:bulk)?(walk|get|getnext)\' .+ \'(udp6?|tcp6?):[^\']+\' \'([^\']+)\'\]/m';
```

Comment (peelman) [reply LibreNMS/Util/ModuleTestHelper.php:194]: @Jellyfrog that looks like a separate bug :)

**Pull request merged**